src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -1049,7 +1049,7 @@ components:
           type: array
           description: A list of the Prisoner's previous qualifications. Can be empty but not null.
           items:
-            $ref: '#/components/schemas/AchievedQualification'
+            $ref: '#/components/schemas/AchievedQualificationResponse'
```

Comment:
This change to the swagger spec is for the type `PreviousQualificationsResponse` and it's property `qualifications`. It is now a list of `AchievedQualificationResponse` rather than `AchievedQualification`.
`AchievedQualificationResponse` already exists as a type in this swagger spec and is basically the same as `AchievedQualification` but with additional reference and audit fields.